### ISSUE TITLE
Screen jumps up when writing/pasting towards bottom of screen

### ISSUE THREAD

**Issue by ReaderGuy42** [chrisgrieser/shimmering-focus#27]
When you paste something or write something with a line break towards the bottom of a page, it jumps up or at least does not move along with the new text.

I tested this with another theme and it does not appear to happen there.

Thanks :)

**Comment by chrisgrieser** [comment]
I don't really follow. Could you do a short screen-recording of the issue?

**Comment by ReaderGuy42** [comment]
https://user-images.githubusercontent.com/71895400/135880405-5707296b-5884-4389-90a3-f73c60002920.mp4

I paste a paragraph and the screen sort of "jumps" up. It's probably rather that it doesn't follow the text which is what I would expect.

**Comment by chrisgrieser** [comment]
Hmm, that is indeed weird, I am not aware of anything I did with the theme that would create this sort of behavior. Also I tried to reproduce it on my machine, but most of the time, I did not have this issue. However, in very rare cases, it indeed scroll up a bit. 🤔 

As I am not really sure how to reproduce this issue / what causes, I am sorry to say that at least for now, I wouldn't know how to fix this. 🙁 

When it occurs more often for you, you could try to experiment a bit. If you can narrow down which conditions exactly cause the issue, it might give me a hint what's root of this.

If any other user of the theme also encounters the issue and could post under what conditions this occurs, this would help a lot!

**Comment by ReaderGuy42** [comment]
https://user-images.githubusercontent.com/71895400/136004553-19041765-662f-4894-994f-3ef9f64db86c.mp4

In this video it's a little clearer what I mean. I press enter to get to the next line, and then when it should scroll down beyond the current screen it jumps up a little bit and then doesn't catch up to the cursor anymore.


https://user-images.githubusercontent.com/71895400/136004944-5ef8ce42-2d37-45c7-a69b-b0e71a3ff347.mp4

This is with another theme ("Minimal") where it works as intended.

**Comment by chrisgrieser** [comment]
yeah, I remember looking into it at some point, but to be frank, I could not find out what causes it :(

**Comment by chrisgrieser** [comment]
No Problem! Leave it open, I am still learning CSS myself and I might find a solution at some point in the future.

Btw, if you like multi-color-highlighting, you should also check out this plugin, which supports even more color: https://github.com/chetachiezikeuzor/Highlightr-Plugin
